TAS_TBDAdapter/CRUD/Read.cs

```diff
@@ -90,13 +90,13 @@ private List<Space> ReadSpaces(List<string> ids = null)
         {
             TBD.Building building = tbdDocument.Building;
 
-            List<Space> spaces = new List<Space>();
+            List <Space> spaces = new List<Space>();
```

Comment:
Delete the space between `List` and `<Space>`

Reply:
done